### DISCUSSION TITLE
How to get commit objects in form of dictionaries

### DISCUSSION THREAD

**Discussion by mayurjs26** [ishepard/pydriller#157] [Q&A] [answered]
Need commit objects in the form of dictionaries in order to use them in multiprocessing. Please help if anyone knows a way to get it done.

**Comment by ishepard** [comment, marked as answer]
Hi @mayurjs26! Sorry I missed this.
Currently you can not obtained the list of commits as a dictionary. You can as a list. 

BTW, hopefully we will implement multiprocessing inside pydriller very soon. So you will be able to do it natively 😄
